lib/node_modules/@stdlib/blas/ext/base/cfill/README.md

```diff
@@ -240,16 +240,15 @@ im = imagf( y );
 <!-- eslint no-undef: "error" -->
 
 ```javascript
-var discreteUniform = require( '@stdlib/random/base/discrete-uniform' );
-var filledarrayBy = require( '@stdlib/array/filled-by' );
+var discreteUniform = require( '@stdlib/random/array/discrete-uniform' );
+var Complex64Array = require( '@stdlib/array/complex64' );
 var Complex64 = require( '@stdlib/complex/float32/ctor' );
 var cfill = require( '@stdlib/blas/ext/base/cfill' );
 
-function rand() {
-    return new Complex64( discreteUniform( 0, 10 ), discreteUniform( -5, 5 ) );
-}
-
-var x = filledarrayBy( 10, 'complex64', rand );
+var xbuf = discreteUniform( 10, -100, 100, {
+    'dtype': 'float32'
+});
+var x = new Complex64Array( xbuf );
```

Comment:
```suggestion
var x = new Complex64Array( xbuf.buffer );
```

Reply:
You need to explicitly provide the `ArrayBuffer`; otherwise, the data is copied.